sdk/tests/deqp/functional/gles3/es3fUniformBlockTests.js

```diff
@@ -393,7 +393,7 @@ goog.scope(function() {
         ];
 
         /** @type {Array<Object.<string, glsUniformBlockCase.UniformFlags>>} */
-        var layoutFlags = [{ name: 'shared', flags: glsUniformBlockCase.UniformFlags.LAYOUT_SHARED },
+        var layoutFlags = [ /* { name: 'shared', flags: glsUniformBlockCase.UniformFlags.LAYOUT_SHARED }, */
```

Comment:
I left it commented out because this code could be potentially useful to somebody.